www-apps/radicale/radicale-3.0.6-r2.ebuild

```diff
@@ -23,50 +23,60 @@ RDEPEND="
 	acct-group/radicale
 	dev-python/defusedxml
 	dev-python/passlib[bcrypt,${PYTHON_USEDEP}]
-	>=dev-python/vobject-0.9.6[${PYTHON_USEDEP}]
-	>=dev-python/python-dateutil-2.8.1[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/vobject[${PYTHON_USEDEP}]
+	dev-python/waitress[${PYTHON_USEDEP}]
 	sys-apps/util-linux
 "
 
+BDEPEND="test? ( ${RDEPEND} )"
+
 S="${WORKDIR}/${MY_P}"
 
-RDIR=/var/lib/${PN}
+RDIR=/var/lib/"${PN}"
 
-python_install_all() {
-	rm README* || die
+DOCS=( DOCUMENTATION.md NEWS.md README.md )
+
+src_prepare() {
+	sed -i '/^addopts =/d' setup.cfg || die
+	distutils-r1_src_prepare
+}
 
+python_install_all() {
 	# init file
 	newinitd "${FILESDIR}"/radicale-r3.init.d radicale
 	systemd_dounit "${FILESDIR}/${PN}.service"
 
 	# directories
-	keepdir ${RDIR}
-	fowners ${PN}:${PN} ${RDIR}
-	fperms 0750 ${RDIR}
+	keepdir "${RDIR}"
+	fperms 0750 "${RDIR}"
+	fowners "${PN}:${PN}" "${RDIR}"
 
 	# config file
-	insinto /etc/${PN}
+	insinto /etc/"${PN}"
 	doins config
 
 	# fcgi and wsgi files
-	exeinto /usr/share/${PN}
+	exeinto /usr/share/"${PN}"
 	doexe radicale.wsgi
 
 	distutils-r1_python_install_all
 }
 
+distutils_enable_tests pytest
+
 pkg_postinst() {
 	local _erdir="${EROOT}${RDIR}"
 
 	einfo "A sample WSGI script has been put into ${EROOT}/usr/share/${PN}."
 	einfo "You will also find there an example FastCGI script."
 	if [[ $(stat --format="%U:%G:%a" "${_erdir}") != "${PN}:${PN}:750" ]]
 	then
-		ewarn "Unsafe file permissions detected on ${_erdir}. This probably comes"
-		ewarn "from an earlier version of this ebuild."
+		echo ""
```

Comment:
https://devmanual.gentoo.org/tools-reference/echo/index.html#abuse-of-echo

Reply:
Fixed